gengokit/httptransport/types.go

```diff
@@ -36,6 +36,9 @@ type Binding struct {
 // useful for templating http transport.
 type Field struct {
 	Name string
+	// PBFieldName 'snake_case' in `protobuf:"varint,1,opt,name=snake_case,json=snakeCase" json:"snake_case,omitempty"`
+	// Where Name is 'SnakeCase'
+	PBFieldName string
```

Comment:
If all we are using this for is the query parameter field name, maybe it should just be something like:
```
QueryFieldName
```